platform-sdk/swirlds-logging/src/test/java/com/swirlds/logging/HandlerLoggingLevelConfigStressTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.logging;
+
+import com.swirlds.base.test.fixtures.concurrent.TestExecutor;
+import com.swirlds.base.test.fixtures.concurrent.WithTestExecutor;
+import com.swirlds.config.api.Configuration;
+import com.swirlds.logging.api.Level;
+import com.swirlds.logging.api.internal.configuration.ConfigLevelConverter;
+import com.swirlds.logging.api.internal.configuration.MarkerStateConverter;
+import com.swirlds.logging.api.internal.level.HandlerLoggingLevelConfig;
+import com.swirlds.test.framework.config.TestConfigBuilder;
+import jakarta.inject.Inject;
+import java.util.List;
+import java.util.stream.IntStream;
+import org.junit.jupiter.api.Test;
+
+@WithTestExecutor
+public class HandlerLoggingLevelConfigStressTest {
+
+    @Inject
+    TestExecutor testExecutor;
+
+    private static Level getRandomLevel() {
```

Comment:
Instead of doing it like that I would prefer a test with params. random values in tests are always bad